### DISCUSSION TITLE
Are there any plans to add support for passkeys and will KPXC continue to work with KDBX x3?

### DISCUSSION THREAD

**Discussion by SuperSentient** [keepassxreboot/keepassxc#8584] [Q&A] [answered]
Since many major sites are moving to passkeys are there any plans to add this to KeepassXC?
Also will KeepassXC continue work with KDBX x3?

**Comment by droidmonkey** [comment, marked as answer]
See #8214. Kdbx3 is severely deprecated, we will likely move to import only in the future.

**Reply by SuperSentient** [reply]
Thanks. Well that kinda sucks no support for passkeys.
And how long down the road would you say that KDBX3 will be deprecated?

**Reply by droidmonkey** [reply]
Maybe for 2.8.0 and import only on 2.9.0

**Reply by SuperSentient** [reply]
Speaking of KDBX I'm running Keepass2.45 on Linux Mint 19 and on Kubuntu 20.04. What version of KDBX does Keepass2.45 use?

**Reply by droidmonkey** [reply]
Probably kdbx 4. I'm not sure why you would not use KeePassXC on Linux. Keepass 2.45 is outdated.

**Reply by SuperSentient** [reply]
I've been asking myself that question for awhile now. I mean when I first installed on my Linux machines it was still receiving updates.
I guess the developer must've gave up or something. But KeepassXC is much better it has more features then Keepass2 does such as the option to auto type just the username or password also its native to Linux it doesn't require Mono also its cross platform.

**Reply by phoerious** [reply]
I don't understand what passkeys has to do with KDBX 3. Don't use it, upgrade to KDBX 4!

**Reply by SuperSentient** [reply]
When is KeepassXC 2.8.0 planned to be released?

**Reply by phoerious** [reply]
2.7.2 is due soon. We have no timeline for 2.8.0.

**Reply by michaelk83** [reply]
@SuperSentient you can check the release history to get a rough idea.

**Reply by SuperSentient** [reply]
Well this morning I installed version 2.7.2 and it worked fine but there was an update today version 2.7.3 and it has an bug it says expires in three days. I already made a bug report. I hope this is fixed soon.
